pybikes/samba.py

```diff
@@ -114,3 +114,59 @@ def update(self, scraper=None):
                 'status': self.get_status(online_status, operation_status)
             }
             self.stations.append(station)
+
+class SambaArgentina(BaseSystem):
+    sync = True
+    _STATIONS_RGX = "var\ estacoes\ =\ (\[.*\]);//]]>"
+
+    def __init__(self, tag, url, meta):
+        super(SambaArgentina, self).__init__(tag, meta)
+        self.feed_url = url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+
+        html_data = scraper.request(self.feed_url)
+        string_data = re.search(SambaArgentina._STATIONS_RGX, html_data).group(1)
+        strs = string_data.replace('[','').split('],')
+        data = [map(str, s.encode('utf-8').replace(']','').split(',')) for s in strs]
+
+        # Each station is like follows
+        # [
+        #   'Pellegrini',
+        #   '-32.953886',
+        #   '-60.656628',
+        #   '',
+        #   'A',
+        #   'EO',
+        #   '9',
+        #   '9',
+        #   '11',
+        #   'Est_Normal 1',
+        #   './img/ico_operacao.png',
+        #   '1',
+        #   'Museo Castagnino',
+        #   'en operación',
+        # ]
+
+        stations = []
+
+        for item in data:
+            name = item[12].replace("'", "")
+            latitude = float(item[1].replace("'", ""))
+            longitude = float(item[2].replace("'", ""))
+            bikes = int(item[7].replace("'", ""))
+            free = int(item[8].replace("'", ""))
+            total_slots = free + bikes
```

Comment:
Unless it's clearly indicated on site, it's better to not infer `total_slots`, since free + bikes might not count broken slots / bikes.